src/Framework/Framework/Compilation/ControlTree/DefaultControlResolver.cs

```diff
@@ -45,6 +46,7 @@ public DefaultControlResolver(DotvvmConfiguration configuration, IControlBuilder
                         startupTracer?.TraceEvent(StartupTracingConstants.InvokeAllStaticConstructorsStarted);
                         InvokeStaticConstructorsOnAllControls();
                         ResolveAllPropertyAliases();
+                        ResolveAllControlAliases();
```

Comment:
I'd slightly prefer if this method returned the `controlNameMappings` dictionary. It would be clearer that nobody can access it before it's actually initialized, or while it's being initialized.